tags/guide/fabricdebug.ytag

```diff
@@ -0,0 +1,17 @@
+type: embed
+
+colour: red
+embed:
+  title: Enabling Debug Logs for Fabric
+
+---
+
+Add `-Dfabric.log.level=debug` to the VM arguments when running in a development environment.
+
+**How to add the arguement to `launch.cfg` (Preferred):**
+Locate the `launch.cfg` file in `.gradle/loom-cache`.
+Under `commonProperties`, add `fabric.log.level=debug` to enable debugging
```

Comment:
No, do not touch this file. It should be added via the IDE, or gradle.

Reply:
oh ok